src/index.js

```diff
@@ -34,12 +42,21 @@ class TodoList {
       checkInput.checked = item.completed;
       checkInput.className = 'todo-list-ul-li-box';
 
+      checkInput.addEventListener('change', (e) => {
+        const textInput = e.target.parentNode.lastChild;
+        toggleCompleteTask(textInput); /* eslint-disable-line no-undef */
```

Comment:
- [OPTIONAL] You can also undo this linter disabling, I found out it has no effect and your code will still run fine.

Reply:
it has been removed...thanks a lot.